src/pint/scripts/photonphase.py

```diff
@@ -227,7 +227,7 @@ def main(argv=None):
 
         # Calculate phases
         log.debug("Evaluating polycos")
-        phases = p.eval_phase(mjds)
+        phases = ptable.eval_phase(mjds)
```

Comment:
@sashalvna can you look at this fix? Without this edit, it was crashing for me and the test was not passing. Do you agree it is correct now?

Reply:
Yes, that is correct! ptable contains the polycos that it should be using to calculate the phases.

Reply:
Thanks much! I can't figure out how it was working before, but it works now!